src/codegeneration/ObjectLiteralTransformer.js

```diff
@@ -54,7 +54,19 @@ class FindAdvancedProperty extends FindVisitor {
     if (isProtoName(tree.name))
       this.protoExpression = tree.value;
     else
-      super.visitPropertyNameAssignment(tree);
+      return super.visitAny(tree.name);
```

Comment:
In case I don't know, I usually use `return` as it always does the right thing whether the function returns void or not.

With proper tail calls it also makes a semantic difference; with return the stack does not grow (not that it matters here, but I think this makes it the better idiom).